aspnetcore/blazor/layouts.md

```diff
@@ -12,6 +12,8 @@ uid: blazor/layouts
 
 By [Rainer Stropek](https://www.timecockpit.com) and [Luke Latham](https://github.com/guardrex)
 
+[!INCLUDE[](~/includes/blazorwasm-preview-notice.md)]
```

Comment:
I think we can remove this one. There isn't anything in this article that is Blazor WebAssembly specific.